docs/4.3/admin-guide.md

```diff
@@ -0,0 +1,2622 @@
+obtain the checksum by adding `.sha256` to the binary.
+
+``` bash
+$ curl https://get.gravitational.com/teleport-v{{ teleport.version }}-darwin-amd64-bin.tar.gz.sha256
+{{ teleport.sha }}  teleport-v{{ teleport.version }}-darwin-amd64-bin.tar.gz
+```
+
+## Definitions
+
+Before diving into configuring and running Teleport, it helps to take a look at
+the [Teleport Architecture](architecture/teleport_architecture_overview.md) and review the key concepts this
+document will be referring to:
+
+|Concept   | Description
+|----------|------------
+|Node      | Synonym to "server" or "computer", something one can "SSH to". A node must be running the [ `teleport` ](cli-docs.md#teleport) daemon with "node" role/service turned on.
+|Certificate Authority (CA) | A pair of public/private keys Teleport uses to manage access. A CA can sign a public key of a user or node, establishing their cluster membership.
+|Teleport Cluster | A Teleport Auth Service contains two CAs. One is used to sign user keys and the other signs node keys. A collection of nodes connected to the same CA is called a "cluster".
+|Cluster Name | Every Teleport cluster must have a name. If a name is not supplied via `teleport.yaml` configuration file, a GUID will be generated.**IMPORTANT:** renaming a cluster invalidates its keys and all certificates it had created.
+|Trusted Cluster | Teleport Auth Service can allow 3rd party users or nodes to connect if their public keys are signed by a trusted CA. A "trusted cluster" is a pair of public keys of the trusted CA. It can be configured via `teleport.yaml` file.
+
+## Teleport Daemon
+
+The Teleport daemon is called [ `teleport` ](cli-docs.md#teleport) and it supports
+the following commands:
+
+|Command     | Description
+|------------|-------------------------------------------------------
+|start       | Starts the Teleport daemon.
+|configure   | Dumps a sample configuration file in YAML format into standard output.
+|version     | Shows the Teleport version.
+|status      | Shows the status of a Teleport connection. This command is only available from inside of an active SSH session.
+|help        | Shows help.
+
+When experimenting, you can quickly start [ `teleport` ](cli-docs.md#teleport)
+with verbose logging by typing [ `teleport start -d` ](cli-docs.md#teleport-start)
+.
+
+!!! danger "WARNING"
+
+    Teleport stores data in `/var/lib/teleport` . Make sure that
+    regular/non-admin users do not have access to this folder on the Auth
+    server.
+
+### Systemd Unit File
+
+In production, we recommend starting teleport daemon via an init system like
+`systemd` . Here's the recommended Teleport service unit file for systemd:
+
+``` yaml
+[Unit]
+Description=Teleport SSH Service
+After=network.target
+
+[Service]
+Type=simple
+Restart=on-failure
+ExecStart=/usr/local/bin/teleport start --config=/etc/teleport.yaml --pid-file=/var/run/teleport.pid
+ExecReload=/bin/kill -HUP $MAINPID
+PIDFile=/var/run/teleport.pid
```

Comment:
Should be `/run` rather than `/var/run` now as that's what `systemd` encourages; it generates warnings for the `/var/run` format.